script/script.js

```diff
@@ -0,0 +1,24 @@
+// Declare input and output
+const ioChatbot = {
+   input : "hello",
+   output : "hi"
+};
+console.log(ioChatbot);
+
+// Reply function for getting input element
+function reply(){
+   const question = document.querySelector("#input");
+   if(question.value === "hello"){
+       document.querySelector("#output").textContent = "hi";
+   } else{
+       document.querySelector("#output").textContent = "I don't understand that command. Please enter another.";
+   }
+
+   console.log(question.value);
+}
+
+if(document.querySelector("#submit")){
+document.querySelector("#submit").addEventListener("click", function(){
```

Comment:
For your callback function, it's not necessary to create an anonymous function (a function with no name, i.e. `function() { ... }`) and call `reply()` inside of it. It works, but you can just pass in the name of your `reply` function directly. Like this:

```javascript
 document.querySelector("#submit").addEventListener("click", reply);
```

Or your callback function can be an anonymous function like below. If you do this, you don't need to write a separate `reply` function because you have included it inline:
```javascript
 document.querySelector("#submit").addEventListener("click", function() {
   const question = document.querySelector("#input");
   if(question.value === "hello"){
       document.querySelector("#output").textContent = "hi";
   } else{
       document.querySelector("#output").textContent = "I don't understand that command. Please enter another.";
   }
});
```